docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -290,6 +290,22 @@ The string argument of the `dayOfWeekFormatter` prop has been replaced in favor
  />
 ```
 
+### Strict typing of the date-related props
+
+All the date-related props are now strictly typed to only accept the date format supported by your adapter
+(`Date` object for `date-fns`, `daysjs.Dayjs` object for `days-js` etc...).
+
+If you run into TypeScript errors such as `DesktopDatePickerProps<Date> error Type 'Date' does not satisfy the constraint 'never'`,
```

Comment:
I made sure we have the error here so that people using `Ctrl + F` can easily find the section in the migration guide.

Reply:
The page has a search function as well.
WDYT about making the section in this page a bit concise? 🤔 
As it currently stands, it seems that there is little point in linking to the `base-concepts` page, because this section contains almost all the same information. 🙈 🤷

Reply:
Whay would you remove from this section?

Reply:
Done :+1: